### PR TITLE
Remove errand-boy process group

### PR DESCRIPTION
**Pull request by millerdev**
The `errand-boy` package is no longer required, and the corresponding supervisor-managed process group must be removed to prevent errors when restarting services at the end of a deploy.

##### ENVIRONMENTS AFFECTED

all

### CODE REVIEW

File: changelog/0037-update-supervisor-confs.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+title: Run command to update Supervisor configurations
+key: update-supervisor-confs
+date: 2020-10-14
+optional_per_env: yes
```
Comment (snopoke) [inline changelog/0037-update-supervisor-confs.yml:4]: is this optional?

File: docs/changelog/0037-update-supervisor-confs.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+[05ebc51b](https://github.com/dimagi/commcare-hq/commit/05ebc51ba4b16a420f0fd64c17ffad89501ea36d)
+
+
+## Change Context
+Run management command to remove unused errand-boy processes.
+
+## Details
+The `errand-boy` package is no longer required. It's corresponding process
+group should be removed to avoid deploy errors when restarting services
+when `errand-boy` is not installed.
```
Comment (snopoke) [inline docs/changelog/0037-update-supervisor-confs.md:21]: this seems to indicate that this is not an optional change

File: docs/changelog/0037-update-supervisor-confs.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+## Steps to update
+Run the following management command to update supervisor configs:
+
+```bash
+cchq <env> --control update-supervisor-confs
```
Comment (snopoke) [inline docs/changelog/0037-update-supervisor-confs.md:27]: I would drop the `--control` since we don't know where the user will run the command.

**Review by snopoke — APPROVED** [review]

File: docs/changelog/0037-update-supervisor-confs.md (hunk 3 above)
Comment (millerdev) [reply docs/changelog/0037-update-supervisor-confs.md:27]: Thanks. https://github.com/dimagi/commcare-cloud/pull/4268/commits/13190aca9f6874a0edd46b14952ca15db4c5af93

File: changelog/0037-update-supervisor-confs.yml (hunk 1 above)
Comment (millerdev) [reply changelog/0037-update-supervisor-confs.yml]: No, thanks. https://github.com/dimagi/commcare-cloud/pull/4268/commits/1e0629c996e7cae15222816f583c7d2f6cfd1f40

File: docs/changelog/0037-update-supervisor-confs.md (hunk 2 above)
Comment (millerdev) [reply docs/changelog/0037-update-supervisor-confs.md:21]: https://github.com/dimagi/commcare-cloud/pull/4268/commits/1e0629c996e7cae15222816f583c7d2f6cfd1f40

**Review by snopoke — APPROVED** [review]
The fact that we're using python2.7 for supervisord seems like an issue (for another PR).

**Pull request merged**